AU2/plugins/custom_plugins/TargetingPlugin.py

```diff
@@ -350,6 +350,15 @@ def update_graph(
         The problem is constraint satisfaction so a Prolog-style "generate-and-test" is sufficient.
         """
 
+        deaths_adj = list(set([d for d in deaths if d in targeting_graph]))
+        visited = []
+        for d in deaths:
+            if d not in deaths_adj or d in visited:
+                print(f"WARNING: {d} has been killed more than once. Skipping this player when updating graph...")

```

Comment:
Do we do print statements? I've been returning Labels for warnings to the umpires (compatibilty for a future html frontend and all that)
Unless you do want this to just be a console warning

Reply:
Valid point.

Reply:
I've got an idea for fixing this. Let's get the fix to prod, and then I'll spin up another PR with some new library functions.